jib-core/src/test/java/com/google/cloud/tools/jib/configuration/BuildConfigurationTest.java

```diff
@@ -246,4 +249,38 @@ public void testBuilder_missingValues() throws IOException {
           ex.getMessage());
     }
   }
+
+  @Test
+  public void testBuilder_digestWarning() throws IOException, InvalidImageReferenceException {
+    EventHandlers mockEventHandlers = Mockito.mock(EventHandlers.class);
+    BuildConfiguration.Builder builder =
+        BuildConfiguration.builder()
+            .setEventHandlers(mockEventHandlers)
+            .setTargetImageConfiguration(
+                ImageConfiguration.builder(Mockito.mock(ImageReference.class)).build())
+            .setBaseImageLayersCacheDirectory(Paths.get("ignored"))
+            .setApplicationLayersCacheDirectory(Paths.get("ignored"))
+            .setExecutorService(MoreExecutors.newDirectExecutorService());
+
+    builder
+        .setBaseImageConfiguration(
+            ImageConfiguration.builder(
+                    ImageReference.parse(
+                        "image@sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"))
+                .build())
+        .build();
+    Mockito.verify(mockEventHandlers, Mockito.never())
+        .dispatch(
+            LogEvent.warn(
+                "Base image 'image:tag' does not use a specific image digest - build may not be reproducible"));
```

Comment:
This won't match anyway because it explicitly states `image:tag`. Is it possible to do `LogEvent.warn(anyString())`?